frontend/src/pages/modelServing/screens/metrics/utils.tsx

```diff
@@ -30,8 +30,8 @@ export const getServerMetricsQueries = (
   server: ServingRuntimeKind,
   currentTimeframe: TimeframeTitle,
 ): { [key in ServerMetricType]: string } => {
-  const namespace = server.metadata.namespace;
-  const name = server.metadata.name;
+  const { namespace } = server.metadata;
+  const { name } = server.metadata;
```

Comment:
nit for future reference. However if the eslint rule cannot enforce this, then there's no point to hold PRs for an update.
```suggestion
  const { name, namespace } = server.metadata;
```